.ruby-version

```diff
@@ -1 +0,0 @@
-2.1.3
```

Comment:
This is actually used by local developers so that their RVM or rbenv knows which version is needed by the project. Specifying the Ruby version in the Gemfile has a separate purpose than specifying it in `.ruby-version`.